DESCRIPTION.md

```diff
@@ -8,6 +8,10 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
 
 # Release Notes
 
+- v3.x.x(TBD)
+  - ..
```

Comment:
Feel free to specify what you think is the next version going to be
```suggestion
- v3.11.1(TBD)
```

Reply:
done, thank you !